### PR TITLE
First step to define guidelines for translators

### PR DESCRIPTION
**Pull request by FransMeerhoff**
A first general guideline for translators is made.

### CODE REVIEW

File: Docs/development/jasp-guideline-translators.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+In this document we keep track of some general guidelines that translators of JASP should follow during the tanslation process. This is an ongoing process and this document should help to maintain consistancy. 
+
+## Weblate
+JASP uses the online localisation tool 'Weblate' for its translations. <br>To contribute to the translations of JASP you need to make an account for Weblate:<br>
+* Open the page https://weblate.org/en-gb/  
+* Press the persons icon  
+* And register your new account in https://hosted.weblate.org/accounts/register/
+<br>
+Once you've registered you're account you can open the JASP project at https://hosted.weblate.org/projects/jasp/
+Here you can find an overview of all the different modules JASP consists of and the translations is contains. Every single module contains two components: an interface component and an R component.
```
Comment (boutinb) [inline Docs/development/jasp-guideline-translators.md]: translations **it** contains

File: Docs/development/jasp-guideline-translators.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* Press the persons icon  
+* And register your new account in https://hosted.weblate.org/accounts/register/
+<br>
+Once you've registered you're account you can open the JASP project at https://hosted.weblate.org/projects/jasp/
+Here you can find an overview of all the different modules JASP consists of and the translations is contains. Every single module contains two components: an interface component and an R component. 
+E.g. the Anova module consists of the jaspANOVA-QML component for the interface and the jaspANOVA-R component for the R part. Pressing on the component show the current translation of the component.<br><br>
+<img src="https://static.jasp-stats.org/images/Weblate-component.png" width="800" height="250" /> <br>For adding a new translation:  
+* Press the Start New Translation button.  
+* Choose your language from the available Languages list.  
+* If your language is not in the list report this to Weblate with the 'Cannot find your language in the list above?' link. <br>
```
Comment (boutinb) [inline Docs/development/jasp-guideline-translators.md]: `in the list report this`
in the list**,** report this

File: Docs/development/jasp-guideline-translators.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+When you select your language from the list you will get an overview of the translated and not yet translated strings.<br><br>
+<img src="https://static.jasp-stats.org/images/Weblate-Choosen-Dutch.png" width="800" height="250" /> <br>
+Selecting the part of strings you want to change will bring you to the form where you do the actual translation work. Be aware that all Weblate users have the same rights filling in this form.<br><br>
+<img src="https://static.jasp-stats.org/images/Weblate-Dutch.png" width="800" height="250" /> <br><br>
+There are a lot of options to be choosen here but all are well described in the online documentaion you can find in the right corner of the page. <br><img src="https://static.jasp-stats.org/images/Weblate-Documentation.png" width="80" height="100" /> <br>The working procedure is now as follows:
```
Comment (boutinb) [inline Docs/development/jasp-guideline-translators.md]: to be **chosen**

File: Docs/development/jasp-guideline-translators.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* Choose your language from the available Languages list.  
+* If your language is not in the list report this to Weblate with the 'Cannot find your language in the list above?' link. <br>
+When you select your language from the list you will get an overview of the translated and not yet translated strings.<br><br>
+<img src="https://static.jasp-stats.org/images/Weblate-Choosen-Dutch.png" width="800" height="250" /> <br>
+Selecting the part of strings you want to change will bring you to the form where you do the actual translation work. Be aware that all Weblate users have the same rights filling in this form.<br><br>
+<img src="https://static.jasp-stats.org/images/Weblate-Dutch.png" width="800" height="250" /> <br><br>
+There are a lot of options to be choosen here but all are well described in the online documentaion you can find in the right corner of the page. <br><img src="https://static.jasp-stats.org/images/Weblate-Documentation.png" width="80" height="100" /> <br>The working procedure is now as follows:  
+*  When you save (Save button) your work your tranlated strings are stored in Weblates repostory. And everybody will see your work.
```
Comment (boutinb) [inline Docs/development/jasp-guideline-translators.md]: `your work your tranlated strings`
your work**,** your translated strings

File: Docs/development/jasp-guideline-translators.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+Selecting the part of strings you want to change will bring you to the form where you do the actual translation work. Be aware that all Weblate users have the same rights filling in this form.<br><br>
+<img src="https://static.jasp-stats.org/images/Weblate-Dutch.png" width="800" height="250" /> <br><br>
+There are a lot of options to be choosen here but all are well described in the online documentaion you can find in the right corner of the page. <br><img src="https://static.jasp-stats.org/images/Weblate-Documentation.png" width="80" height="100" /> <br>The working procedure is now as follows:  
+*  When you save (Save button) your work your tranlated strings are stored in Weblates repostory. And everybody will see your work.  
+*  Once you have commited your work (the Manage/Commit button in previous screen) a PR (pull request) is made to the JASP github repository belonging to this component.  For instance see the [JASP Anova repository](https://github.com/jasp-stats/jaspAnova) for the JASP Anova module repository.  
+*  This PR is then merged into this JASP module repository by the responsable developer of ths repository.
```
Comment (boutinb) [inline Docs/development/jasp-guideline-translators.md]: `of ths repository.  `
of this repository.

File: Docs/development/jasp-guideline-translators.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+<img src="https://static.jasp-stats.org/images/Weblate-Dutch.png" width="800" height="250" /> <br><br>
+There are a lot of options to be choosen here but all are well described in the online documentaion you can find in the right corner of the page. <br><img src="https://static.jasp-stats.org/images/Weblate-Documentation.png" width="80" height="100" /> <br>The working procedure is now as follows:  
+*  When you save (Save button) your work your tranlated strings are stored in Weblates repostory. And everybody will see your work.  
+*  Once you have commited your work (the Manage/Commit button in previous screen) a PR (pull request) is made to the JASP github repository belonging to this component.  For instance see the [JASP Anova repository](https://github.com/jasp-stats/jaspAnova) for the JASP Anova module repository.  
+*  This PR is then merged into this JASP module repository by the responsable developer of ths repository.  
+*  During the nightly build new JASP binary translanlation files (.qm and .mo) are generated. These files are added to the setup of JASP, so the new translations should be available in this nighly made version.
```
Comment (boutinb) [inline Docs/development/jasp-guideline-translators.md]: binary **translation** files

Comment (boutinb) [reply Docs/development/jasp-guideline-translators.md]: Add the link to the nightly build

File: Docs/development/jasp-guideline-translators.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+*  This PR is then merged into this JASP module repository by the responsable developer of ths repository.  
+*  During the nightly build new JASP binary translanlation files (.qm and .mo) are generated. These files are added to the setup of JASP, so the new translations should be available in this nighly made version.  
+*  Every week new JASP .po files are generated containing the new translatable string inside JASP. They wil then be merged into Weblate. Some component may then have new strings to translate. 
+
+## Translation of statistical terms
+1. Weblate has the functionality of defining glossaries. Each project can have an assigned glossary for any language as a shorthand for storing terminology. Consistency is more easily maintained this way. If a term is defined in the Weblate glossery in its proper context this definition should be used. Definitions defined in the glossery can be exported and imported.
+2. When in doubt, please use the statistical terms as they occur in introductory statistics textbooks and courses in your language. We would like users to recognize the terms that are used. If you deem this useful you can mention alternative terms in the help file.
```
Comment (boutinb) [inline Docs/development/jasp-guideline-translators.md]: `If you deem this useful you can mention alternative terms in the help file.`
If you find alternative terms useful, you can mention them in the help file. 
_(help file: do you mean the glossary?)_

Comment (boutinb) [reply Docs/development/jasp-guideline-translators.md]: The automatic translation feature of WelLate should be also mentioned here.

File: Docs/development/jasp-guideline-translators.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+2. When in doubt, please use the statistical terms as they occur in introductory statistics textbooks and courses in your language. We would like users to recognize the terms that are used. If you deem this useful you can mention alternative terms in the help file. 
+
+## Capitalization
+As a general rule we use the following conventions. For names of titles, groups and sections all the words are capitalized. For all the subnames in those groups, e.g. the labels of the controls, those names only start with a capital. For instanca the Visual Moduling module is named 'Visual Modeling' and one of the analysis is called 'Generalized Linear Modeling'. But if we look at the section 'Results Display' the checkbox 'Model plot' only starts wit a capital.
+
+
+## Formal or Familiar 
+In JASP we have choosen for a formal way of addressing a user. In some languages there more levels of speech but in general we use the formal level of speech. So in Dutch we use the persons forms 'U' instead of 'jij' and in French 'vous' instead of 'tu'.
```
Comment (boutinb) [inline Docs/development/jasp-guideline-translators.md]: `there more levels`
there are more levels

**Review by boutinb — CHANGES_REQUESTED** [review]

File: Docs/development/jasp-guideline-translators.md (hunk 1 above)
Comment (FransMeerhoff) [reply Docs/development/jasp-guideline-translators.md]: OK

File: Docs/development/jasp-guideline-translators.md (hunk 2 above)
Comment (FransMeerhoff) [reply Docs/development/jasp-guideline-translators.md]: OK

File: Docs/development/jasp-guideline-translators.md (hunk 3 above)
Comment (FransMeerhoff) [reply Docs/development/jasp-guideline-translators.md]: OK

File: Docs/development/jasp-guideline-translators.md (hunk 6 above)
Comment (FransMeerhoff) [reply Docs/development/jasp-guideline-translators.md]: OK

File: Docs/development/jasp-guideline-translators.md (hunk 4 above)
Comment (FransMeerhoff) [reply Docs/development/jasp-guideline-translators.md]: OK

File: Docs/development/jasp-guideline-translators.md (hunk 5 above)
Comment (FransMeerhoff) [reply Docs/development/jasp-guideline-translators.md]: Ok

**Review by boutinb — APPROVED** [review]

**Pull request merged**